Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -195,7 +201,9 @@ private void CreateRange()
             else
             {
                 this.lower = limits[0];
-                this.upper = limits[1];
+                this.upper = limits[1] > Int32.MaxValue
+                    ? limits[1]
+                    : limits[1] + 1;

```

Comment:
This change is what will allow the upper limit to be included in the call to Random.Next for Int32 ranges.  This is my fix for Issue #413.